ddtrace/appsec/_iast/taint_sinks/command_injection.py

```diff
@@ -27,47 +24,16 @@ def get_version() -> str:
 
 
 def patch():
-    if not asm_config._iast_enabled:
-        return
-
-    if not getattr(os, "_datadog_cmdi_patch", False):
-        # all os.spawn* variants eventually use this one:
-        try_wrap_function_wrapper("os", "_spawnvef", _iast_cmdi_osspawn)
-
-    if not getattr(subprocess, "_datadog_cmdi_patch", False):
-        try_wrap_function_wrapper("subprocess", "Popen.__init__", _iast_cmdi_subprocess_init)
-
-        os._datadog_cmdi_patch = True
-        subprocess._datadog_cmdi_patch = True
-
-    _set_metric_iast_instrumented_sink(VULN_CMDI)
+    if asm_config._iast_enabled:
+        subprocess_patch.patch()
+        subprocess_patch.add_str_callback("iast cmdi", _iast_report_cmdi)
```

Comment:
nit: maybe we could use constants instead of literals to make sure we don't have a typo anywhere?